api/permissions.py

```diff
@@ -17,3 +17,10 @@ def has_permission(self, request, view):
 
     def has_object_permission(self, request, view, obj):
         return request.user == obj.user
+
+
+class IsOwnerOrReadOnly(permissions.IsAuthenticated):
```

Comment:
We want them to be able to edit it but not delete it.

Reply:
Oh whoops I forgot this disabled that.